scripts/check-remote-branch.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+set -euo pipefail
+
+function fail { >&2 echo -e "ERROR: $1"; exit 1; }
```

Comment:
Although currently only one function is being used across many scripts (e.g., `fail`), maybe it is worth having a `common.sh` here?
And we could add another function to check the number of arguments and display a custom message if it is not given.

Reply:
Well, maybe, but at the point where you have to do that I'm usually starting to seriously consider just rewriting the whole thing in Python :) Even in the main repo the shenanigans with `common.sh` are pretty brittle and more of a "lesser evil" thing.

Here I already did extract the bigger shared parts into a shared script (`pull-and-resolve-english-changes.sh` - originally the sync PR script was completely independent from it) so I'd rather leave it at that.

Reply:
Yeah, python is a perfect candidate here indeed. So it is fine, just fix the conflicts and we merge it :)